item-spec/item-spec.md

```diff
@@ -106,13 +106,20 @@ A more complete list of possible 'rel' types can be seen at the [IANA page of Li
 that comes along for that. But the derived_from field is seen as a way to encourage fuller specs and at least start a linking
 structure that can be used as a jumping off point for more experiments in provenance tracking*
 
-#### Relative vs Absolute links
+#### Absolute URIs vs Relative References
 
-Currently the JSON schema for links does not require a URI formatting, to give the option for
-implementors to provide relative links. In general, Catalog API's should aim for absolute links
+Currently the JSON schema for links does not require an absolute URI formatting such as
+`http://example.com/foo/bar.json`, to give the option for
+implementors to provide relative references such as `/foo/bar.json`.
+
+In general, Catalog API's should aim for absolute URIs
 whenever possible. But Static Catalogs are potentially more portable if they can be implemented with
-relative links, so that every link doesn't need to be rewritten when the data is copied. The `self`
-link is required to be absolute.
+relative references, so that every link doesn't need to be rewritten when the data is copied to
+a different server, for instance.
+
+It is however required for all relative references to use absolute paths, that is, the references
```

Comment:
They are still relative to something, either the root directory of the file system or the root catalog folder. It needs to be clarified what the root directory is.

Reply:
Maybe [this](https://tools.ietf.org/html/rfc3986#section-4.2)?

Reply:
Yes, you could still navigate through the links by generating 'relative' links comparing self and the other references, but I agree that this is cumbersome and not standard.

Note however that absolute paths are allowed as part of URIs, so if we say nothing I think they would be allowed.. it's ironic but maybe the conclusion of this exercise will be to __forbid__ absolute paths at all, except for self when data is offline or the provider is not confident to consider an absolute URL as a permalink.